.editorconfig

```diff
@@ -0,0 +1,19 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[Makefile]
```

Comment:
This seems to contradict https://github.com/tendermint/coding/blob/master/Coding_Standard.md#non-golang-code

which requests 4 spaces per tab.  which is also what rigel is doing... reformatting all my code.